Examples/UIComponentExample/Examples/Complex layout/Components/Cell.swift

```diff
@@ -0,0 +1,38 @@
+//
+//  Cell.swift
+//  UIComponentExample
+//
+//  Created by y H on 2021/7/21.
+//  Copyright © 2021 Luke Zhao. All rights reserved.
+//
+
+import UIComponent
+import UIKit.UIScreen
+
+struct Cell: ComponentBuilder {
+  let data: Context
+  func build() -> Component {
+    HStack(spacing: 10, alignItems: .center) {
+      VStack(spacing: 5, alignItems: .center) {
+        Image(systemName: "hand.tap")
+        Text("Tap delete", font: .systemFont(ofSize: 10))
+      }
+      Space().size(width: .aspectPercentage(1), height: .fill).inset(10).styleColor(.systemBlue)
+      VStack(justifyContent: .spaceAround) {
+        Text("This can occupy one line").numberOfLines(2).flex()
+        HStack(spacing: 5, alignItems: .center) {
+          Text("long long long long long Text", font: .systemFont(ofSize: 14)).textColor(.secondaryLabel).numberOfLines(1).flex()
+          Image(systemName: "checkmark.shield.fill")
+        }.flex()
+      }.flex()
+    }.inset(10).size(width: UIScreen.main.bounds.width - 80, height: 100).styleColor(data.fillColor).id(data.id)
+  }
+  
+}
+
+extension Cell {
+  struct Context {
+    var fillColor: UIColor
+    var id: String
```

Comment:
Might be good to conform to `Identifiable`. Put `id` at the top. use `let` if possible. You can also use `UUID` to make it easier to create.
```swift
struct Context: Identifiable  {
  let id = UUID()
  let fillColor: UIColor
}
```

Reply:
✅